fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1517,6 +1518,87 @@ ACTOR Future<Void> monitorRemoteCommitted(ProxyCommitData* self) {
 	}
 }
 
+
+ACTOR Future<Void>
+proxySnapCreate(ProxySnapRequest snapReq, ProxyCommitData* commitData)
+{
+	TraceEvent("SnapMasterProxy.SnapReqEnter")
+		.detail("SnapPayload", snapReq.snapPayload)
+		.detail("SnapUID", snapReq.snapUID);
+	try {
+		// whitelist check
+		state ExecCmdValueString execArg(snapReq.snapPayload);
+		state StringRef binPath = execArg.getBinaryPath();
```

Comment:
```suggestion
		StringRef binPath = execArg.getBinaryPath();
```
This doesn't need to be a state variable